src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1,3 +1,5 @@
 databaseChangeLog:
   - include:
       file: db/changelog/changes/01-create-books-table.yaml
+  - include:
+      file: db/changelog/changes/02-create-users-table.yaml
```

Comment:
```suggestion
      file: db/changelog/changes/02-create-users-table.yaml
      
```

Reply:
not resolved, please add an empty new line at the end of file